core/src/main/java/org/keycloak/representations/overrides/NoSubGroupsGroupRepresentation.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.representations.overrides;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class NoSubGroupsGroupRepresentation {
+    protected String id;
+    protected String name;
+    protected String path;
+    protected Map<String, List<String>>  attributes;
+    protected List<String> realmRoles;
+    protected Map<String, List<String>> clientRoles;
+    // TODO: eventually generate code for Nth levels of depth
+    // protected List<GroupRepresentation> subGroups;
```

Comment:
@stianst here we can decide how many level of `subGroups` we support in the CRD, at this point is 1, but I would follow up with any number seems reasonable.